### ISSUE TITLE
[BUG/ISSUE] Incorrect grid definitions in HEMCO standalone

### ISSUE THREAD

**Issue by ltmurray** [geoschem/HEMCO#71]
The grid definitions for 2x2.5, 0.5x0.625 and 0.25x0.3125 in the HEMCO standalone run directory generation are incorrect. @christophkeller please correct me if I am wrong, but XMIN and XMAX should be the westernmost and easternmost edges, and the native GEOS resolutions still have half-polar boxes.

Therefore, HEMCO_sa_Grid.2x25.rc should read (changes emphasized with ** **)
```
XMIN: **-181.25**
XMAX:  **178.75**
YMIN: -90.0
YMAX:  90.0
NX: 144
NY: 91
NZ: 47
YMID: -89.5 -88.0 -86.0 -84.0 -82.0 -80.0 -78.0 -76.0 -74.0 -72.0 -70.0 -68.0 -66.0 -64.0 -62.0 -60.0 -58.0 -56.0 -54.0 -52.0 -50.0 -48.0 -46.0 -44.0 -42.0 -40.0 -38.0 -36.0 -34.0 -32.0 -30.0 -28.0 -26.0 -24.0 -22.0 -20.0 -18.0 -16.0 -14.0 -12.0 -10.0 -8.0 -6.0 -4.0 -2.0
 0.0 2.0 4.0 6.0 8.0 10.0 12.0 14.0 16.0 18.0 20.0 22.0 24.0 26.0 28.0 30.0 32.0 34.0 36.0 38.0 40.0 42.0 44.0 46.0 48.0 50.0 52.0 54.0 56.0 58.0 60.0 62.0 64.0 66.0 68.0 70.0 72.0 74.0 76.0 78.0 80.0 82.0 84.0 86.0 88.0 89.5
```

And, similarly, the XMIN and XMAX for 0.625 and 0.3125 should be -180.25/179.75 and -180.125/179.875.  And the YMID and YEDGE will need to be manually defined for these runs as well, because of the half-polar boxes.

**Comment by yantosca** [comment]
No worries @ltmurray, we can transfer the issue.

**Comment by msulprizio** [comment]
Thanks for pointing this out @ltmurray and sorry for the delayed response. We can fix this in GEOS-Chem 13.1.0.

**Comment by msulprizio** [comment]
This is now fixed in the `dev` branch (commit https://github.com/geoschem/HEMCO/commit/012d251c5e566039593377c937aa9928aa15a6b4) and will be included in GEOS-Chem 13.1.0 / HEMCO 3.0.0.
